azurerm/internal/services/netapp/resource_arm_netapp_volume.go

```diff
@@ -114,22 +129,25 @@ func resourceArmNetAppVolume() *schema.Resource {
 								ValidateFunc: validate.CIDR,
 							},
 						},
-						"cifs_enabled": {
-							Type:     schema.TypeBool,
-							Required: true,
-						},
-						"nfsv3_enabled": {
-							Type:     schema.TypeBool,
-							Required: true,
-						},
-						"nfsv4_enabled": {
-							Type:     schema.TypeBool,
+
+						"protocols_enabled": {
+							Type:     schema.TypeList,
 							Required: true,
+							MaxItems: 1,
+							MinItems: 1,
```

Comment:
You cannot set more then 1 per export?

Reply:
Good catch, yes we can do 2 but same validations applies as I mentioned before.

Reply:
👍